hawc/apps/animal/templates/animal/animalgroup_detail.html

```diff
@@ -1,32 +1,33 @@
 {% extends 'assessment-rooted.html' %}
 
 {% load hawc %}
+{% load bs4 %}
 
 {% block content %}
     {% if obj_perms.edit %}
-      <div class="dropdown btn-group float-right">
-        <a class="btn btn-primary dropdown-toggle" data-toggle="dropdown">Actions</a>
-        <div class="dropdown-menu dropdown-menu-right">
-            <span class="dropdown-header">Animal Group Editing</span>
-            <a class="dropdown-item" href="{% url 'animal:animal_group_update' object.pk %}">Update</a>
-            <a class="dropdown-item" href="{% audit_url object %}">Change logs</a>
-            <a class="dropdown-item" href="{% url 'animal:animal_group_delete' object.pk %}">Delete</a>
-
-            {% if object.dosing_regime and object == object.dosing_regime.dosed_animals %}
+    <div class="d-flex">
+      <h2 class="d-inline-block mb-0">{{object}}</h2>
+      {% actions %}
```

Comment:
looks good. a minor request is to just fix the indentation here to keep it consistent